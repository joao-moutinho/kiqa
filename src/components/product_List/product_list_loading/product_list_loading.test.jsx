import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { ProductListLoading } from ".";


describe('render', () => {

    test('render', () =>{
        const {container} = render(<BrowserRouter> <ProductListLoading /> </BrowserRouter>)
        expect(container).toMatchSnapshot();
    })
})