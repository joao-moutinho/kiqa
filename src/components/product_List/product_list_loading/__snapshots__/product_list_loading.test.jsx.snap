// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render render 1`] = `
<div>
   
  <div
    class="product-list-loading"
  >
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
    <div
      class="product-list-loading-box"
    />
  </div>
   
</div>
`;
